src/Microsoft.Health.Fhir.Web/DevelopmentIdentityProviderRegistrationExtensions.cs

```diff
@@ -121,7 +121,8 @@ public static IConfigurationBuilder AddDevelopmentAuthEnvironment(this IConfigur
                 return configurationBuilder;
             }
 
-            testEnvironmentFilePath = Path.GetFullPath(testEnvironmentFilePath);
+            string normalizedPath = testEnvironmentFilePath.Replace('\\', Path.DirectorySeparatorChar);
```

Comment:
There's not a huge need for a new variable here, especially since we're already transforming it below. We should be able to do something like the following instead. 

```suggestion
            testEnvironmentFilePath = testEnvironmentFilePath.Replace('\\', Path.DirectorySeparatorChar);
            testEnvironmentFilePath = Path.GetFullPath(testEnvironmentFilePath);
```

Also, could probably just be done on one line, but readability may suffer.

Reply:
Thanks for the input. Good points - I agree. I removed the extra variable and ended up one-lining this.